README.md

```diff
@@ -19,15 +19,36 @@ git clone https://github.com/hzeller/timg.git
 cd timg/src
 sudo apt-get install libwebp-dev libgraphicsmagick++-dev    # required libs.
 
-# If you want to include video decoding
+# If you do not want to include video decoding
+make WITH_VIDEO_DECODING=0
+
+# If you want to include video decoding, install these additional libraries
 sudo apt-get install pkg-config libavcodec-dev libavformat-dev libswscale-dev
+make
 
-make WITH_VIDEO_DECODING=1
 sudo make install
 ```
 
 This installs the binary and the [manpage](man/timg.1.md).
 
+#### Install on macOS
+Assuming homebrew is available to install required dependencies
+```
+git clone https://github.com/hzeller/timg.git
+cd timg/src
+
```

Comment:
Maybe to avoid repeating all the things, we could put the clone part in a separate section

### Install

First, get the repository
```bash
git clone ...
cd timg/src
```

#### Compiling on Debian/Ubuntu
```bash
git clone
...
```

#### Compiling on MacOS
```bash
brew instal ...
...
```